src/Ormolu/Parser.hs

```diff
@@ -191,3 +193,5 @@ setDefaultExts flags = foldl' GHC.xopt_set flags autoExts
     allExts = [minBound..maxBound]
 
 deriving instance Bounded Extension
+
+-- Semicolon Warnings
```

Comment:
This seems to be just a "header" at the very end of the file? Should be dropped as well, right?